src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -744,22 +744,20 @@ protected function fetchTableOptionsByTable(string $databaseName, ?string $table
     private function buildQueryConditions($tableName): array
     {
         $conditions = [];
-        $schemaName = null;
 
         if ($tableName !== null) {
             if (strpos($tableName, '.') !== false) {
                 [$schemaName, $tableName] = explode('.', $tableName);
+                $conditions[]             = 'n.nspname = ' . $this->_platform->quoteStringLiteral($schemaName);
+            } else {
+                $conditions[] = 'n.nspname = ANY(current_schemas(false))';
```

Comment:
This logic refers from https://github.com/doctrine/dbal/blob/3.4.x/src/Platforms/PostgreSQLPlatform.php#L424-L429